CIP-1694/README.md

```diff
@@ -488,7 +491,7 @@ Depending on the type of governance action, an action will thus be ratified when
 
 * the constitutional committee approves of the action (the number of members who vote `Yes` meets the threshold of the constitutional committee)
 * the DReps approve of the action (the stake controlled by the DReps who vote `Yes` meets a certain threshold of the total active voting stake)
-* the SPOs approve of the action (the stake controlled by the SPOs who vote `Yes` meets a certain threshold over the total delegated active stake for the epoch)
+* the SPOs approve of the action (the stake controlled by the SPOs who vote `Yes` meets a certain threshold of the total active voting stake, excepting Hard Fork Governance Actions)
```

Comment:
The total active voting stake usually doesn’t include 'Auto-Abstain' in the calculations for any governance actions, including hard fork governance. Since hard forks will not include 'Auto-Abstain' and 'No-Confidence' voting options, wouldn’t it be better to rephrase this with the assumption that we are effectively discussing active stake delegated to stake pools, rather than just the active stake that has voted?